libdill.h

```diff
@@ -967,6 +967,75 @@ DILL_EXPORT int dill_ws_response_key(
 #define ws_response_key dill_ws_response_key
 #endif
 
+/******************************************************************************/
+/*  SOCKS5                                                                    */
+/******************************************************************************/
+
+// SOCKS5 client commands
+#define DILL_SOCKS5_CONNECT (0x01)
```

Comment:
These doesn't seem to be needed in the header file.

Reply:
SOCKS5_CONNECT, SOCKS5_BIND, SOCKS5_UDP_ASSOCIATE are the possible successful return values of socks5_proxy_recvcommand[byname]